Fix shellcheck issues

Please fix all [shellcheck](https://www.shellcheck.net/) issues:

```
➜ shellcheck get-blackduck-report.sh

In get-blackduck-report.sh line 161:
if [ sbom_type == "CYCLONEDX_13" || sbom_type == "CYCLONEDX_14" ]
               ^-- SC2050 (warning): This expression is constant. Did you forget the $ on a variable?
                                 ^-- SC2109 (error): Instead of [ a || b ], use [ a ] || [ b ].
                                              ^-- SC2050 (warning): This expression is constant. Did you forget the $ on a variable?


In get-blackduck-report.sh line 163:
  echo "INFO: sbomType "CYCLONEDX_13" or "CYCLONEDX_14" allows reportFormat values of "JSON"."
                        ^----------^ SC2140 (warning): Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                          ^----------^ SC2140 (warning): Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                                                                       ^--^ SC2140 (warning): Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?


In get-blackduck-report.sh line 210:
echo "sbom-file=report.zip" >> $GITHUB_OUTPUT
                               ^------------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
echo "sbom-file=report.zip" >> "$GITHUB_OUTPUT"


In get-blackduck-report.sh line 211:
echo "sbom-contents=${report_contents}" >> $GITHUB_OUTPUT
                                           ^------------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
echo "sbom-contents=${report_contents}" >> "$GITHUB_OUTPUT"

For more information:
  https://www.shellcheck.net/wiki/SC2109 -- Instead of [ a || b ], use [ a ] ...
  https://www.shellcheck.net/wiki/SC2050 -- This expression is constant. Did ...
  https://www.shellcheck.net/wiki/SC2140 -- Word is of the form "A"B"C" (B in...
```

In particular [SC2050](https://www.shellcheck.net/wiki/SC2050) on line 161 looks like a real bug (2x).

--- a/get-blackduck-report.sh
+++ b/get-blackduck-report.sh
@@ -158,9 +158,9 @@ version=$4
 
 sbom_type=${5:-"SPDX_22"}
 
-if [ sbom_type == "CYCLONEDX_13" || sbom_type == "CYCLONEDX_14" ]
+if [ "$sbom_type" == "CYCLONEDX_13" ] || [ "$sbom_type" == "CYCLONEDX_14" ]
 then
-  echo "INFO: sbomType "CYCLONEDX_13" or "CYCLONEDX_14" allows reportFormat values of "JSON"."
+  echo "INFO: sbomType \"CYCLONEDX_13\" or \"CYCLONEDX_14\" allows reportFormat values of \"JSON\"."
   report_format="JSON"
 else
   report_format=${6:-"JSON"}
@@ -207,5 +207,5 @@ report_contents=$(get_report_contents)
 echo "| got content information"
 echo
 
-echo "sbom-file=report.zip" >> $GITHUB_OUTPUT
-echo "sbom-contents=${report_contents}" >> $GITHUB_OUTPUT
+echo "sbom-file=report.zip" >> "$GITHUB_OUTPUT"
+echo "sbom-contents=${report_contents}" >> "$GITHUB_OUTPUT"

